src/components/referrals/ReferralsComponentLabel.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <li class="documentBox">
+        <font-awesome-icon :icon="classOfIcon" />
+        <h5 class="documentBox__title">{{ name }} {{ date }}</h5>
+        <button-component
+            small
+            @click="openModalDetails = true">
+            Szczegóły
+        </button-component>
+        <teleport to="body">
+            <transition name="modal">
+                <div
+                    v-if="openModalDetails"
+                    class="modal">
+                    <modal-component
+                        ref="modalDetils"
+                        @close="openModalDetails = false">
+                        <referrals-details
+                            v-bind:key="getPatientRefferals[index].id"
+                            :names="getPatientRefferals[index].doctorData"
+                            :dates="getPatientRefferals[index].date"
+                            :specializations="
+                                getPatientRefferals[index].specializationType
+                            "
+                            :typeOfTests="
+                                getPatientRefferals[index].testType
+                            " />
+                    </modal-component>
+                </div>
+            </transition>
+        </teleport>
+    </li>
+</template>
+
+<script setup>
+    import { computed, ref } from 'vue';
+    import { storeToRefs } from 'pinia';
+    import { onClickOutside } from '@vueuse/core';
+    import ButtonComponent from '../ButtonComponent.vue';
+    import ReferralsDetails from './ReferralsDetails.vue';
+    import ModalComponent from '../ModalComponent.vue';
+    import useUserStore from '../../stores/userStore';
+
+    const openModalDetails = ref(null);
+    const modalDetils = ref(null);
+    const userStore = useUserStore();
+    const { getPatientRefferals } = storeToRefs(userStore);
+
+    onClickOutside(modalDetils, () => (openModalDetails.value = false));
+
+    defineProps({
+        name: {
+            type: String,
+            default: 'Lekarz',
+        },
+        date: {
+            type: String,
+            default: '',
+        },
+        id: {
+            type: String,
+            default: null,
+        },
+        typeOfDoc: {
+            type: String,
+            default: 'referralView',
```

Comment:
What does mean this default value?

Reply:
now i thing that all typeOfDoc is unnecesarry

Reply:
i dont use it, so i will delete it.